src/kokkos_ext/ArborX_DetailsKokkosExtViewHelpers.hpp

```diff
@@ -55,32 +55,17 @@ void reallocWithoutInitializing(ExecutionSpace const &space, View &v,
                                 size_t n7 = KOKKOS_IMPL_CTOR_DEFAULT_ARG)
 {
   static_assert(Kokkos::is_execution_space<ExecutionSpace>::value);
-  static_assert(Kokkos::is_view<View>::value);
-  static_assert(View::is_managed, "Can only realloc managed views");
-
-  size_t new_extents[8] = {n0, n1, n2, n3, n4, n5, n6, n7};
-  bool has_requested_extents = true;
-  for (unsigned int dim = 0; dim < v.rank_dynamic; ++dim)
-    if (new_extents[dim] != v.extent(dim))
-    {
-      has_requested_extents = false;
-      break;
-    }
-
-  if (!has_requested_extents)
-    v = View(Kokkos::view_alloc(space, Kokkos::WithoutInitializing, v.label()),
-             n0, n1, n2, n3, n4, n5, n6, n7);
+  Kokkos::realloc(Kokkos::view_alloc(space, Kokkos::WithoutInitializing), v, n0,
+                  n1, n2, n3, n4, n5, n6, n7);
```

Comment:
Should we get rid of `reallocWithoutInitializing` completely? It's an empty function.

Reply:
Yes, I avoided doing that so far thinking that someone would care about the shorter name.